src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml.cs

```diff
@@ -40,6 +49,32 @@ public PopupWindow(PopupWindowViewModel viewModel, HostControlInfo hInfo)
             ContentRichTextBox.Width = popupViewModel.Width - (popupBordersOffSet * 4);
             HorizontalOffset = hInfo.HorizontalPopupOffSet;
             VerticalOffset = hInfo.VerticalPopupOffSet;
+
+            if (!string.IsNullOrEmpty(hInfo.HtmlPage))
+            {
+                ContentRichTextBox.Visibility = Visibility.Hidden;
+                Opened += PopupWindow_Opened;
+            }
+
+            Closed += PopupWindow_Closed;
+        }
+
+        ~PopupWindow()
+        {
+            Opened -= PopupWindow_Opened;
+            Closed -= PopupWindow_Closed;
+        }
+
+        private void PopupWindow_Closed(object sender, EventArgs e)
+        {
+            if(webBrowserWindow != null)
```

Comment:
for example, now is a good time to clear the close event subscription, since the window just closed.